src/main/java/com/mateacademy/table/Skill.java

```diff
@@ -0,0 +1,27 @@
+package com.mateacademy.table;
+
+import com.mateacademy.listener.MyListener;
+import lombok.*;
+
+import javax.persistence.*;
+
+@Entity
+@Getter
+@Setter
+@ToString
+@NoArgsConstructor
+@AllArgsConstructor
+@EntityListeners(MyListener.class)
+@javax.persistence.Table(name = "skill")
+public class Skill extends TableX {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @Column(name = "area")
+    private String area;
+
+    @Column(name = "skill_level")
+    private String skillsLevel;
```

Comment:
For "junior" you can use "j", "jun", "junior" or something like these. How can we be more specific?